src/components/note-containers/note-containers.js

```diff
@@ -3,17 +3,37 @@ import './note-containers.css';
 import TodoData from "../../service/service";
 
 export default class NoteContainers extends Component {
+    
+    todoData = new TodoData();
 
-   
+    state = {
+        notes: []
+    };
```

Comment:
there is no property declaration syntax in es6, probably it works because of webpack + babel
you need to use constructor to define fields/properties of the class
https://stackoverflow.com/questions/38269083/declare-a-class-property-outside-of-a-class-method

Look on first example here
https://daveceddia.com/where-initialize-state-react/